notebook.ipynb

```diff
@@ -2,7 +2,15 @@
  "cells": [
```

Comment:
<ul><li>Acronym API needs to be spelled out on first mention.</li><li>No citation or link to U-Net model's paper. Also inconsistent hyphenation of U-Net vs UNet.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/67a1e320-7c47-4ea9-8df8-e868326bc90b/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>* Added definitions section</p><p>* Added link to original U-Net paper, and updated to be consistent with the original paper - U-Net.</p>